chemprop/args.py

```diff
@@ -702,7 +760,12 @@ def process_args(self) -> None:
                     raise ValueError(f'Additional features were provided using the argument {features_argument}. The same kinds of features must be provided for the separate validation set.')
                 if self.separate_test_path is not None and test_features_path is None:
                     raise ValueError(f'Additional features were provided using the argument {features_argument}. The same kinds of features must be provided for the separate test set.')
```

Comment:
You should add the `--separate_val_constraints_path` and `--separate_test_constraints_path` to the checks in L751-762.